src/streamlit_extras/image_selector/__init__.py

```diff
@@ -0,0 +1,136 @@
+from io import BytesIO
+from typing import Literal
+
+import numpy as np
+import plotly.graph_objects as go
+import requests
+import streamlit as st
+from PIL import Image, ImageDraw
+
+from streamlit_extras import extra
+
+
+@extra
+def image_selector(
+    image: Image.Image,
```

Comment:
*suggesiton:* It would be nice to make this a bit more flexible, and handle image strings as well as Image objects. I think that would make people a lot more likely to use it, and it would be easy to just go ahead and convert to a PIL image as-needed.